source/draw/RasterCanvas.ooc

```diff
@@ -71,7 +71,13 @@ RasterCanvas: abstract class extends Canvas {
 	_map: func (point: IntPoint2D) -> IntPoint2D {
 		point + this _size / 2
 	}
-	resizeNearestNeighbour: static func <T> (sourceBuffer, resultBuffer: T*, sourceBox, resultBox: IntBox2D, sourceStride, resultStride, bytesPerPixel: Int) {
+	_resizePacked: func <T> (sourceBuffer, resultBuffer: T*, sourceBox, resultBox: IntBox2D, sourceStride, resultStride, bytesPerPixel: Int) {
```

Comment:
And I can't parametrize this one on the subclass of `RasterImage`. `T` still means `pixel format` here, unfortunately.
Anyway I think it's better than nothing, imagine code for linear and bilinear interpolation repeated in monochrome, uv, yuv, bgr and bgra classes...